x-pack/plugins/security_solution/public/common/lib/kibana/kibana_react.mock.ts

```diff
@@ -150,6 +150,9 @@ export const createStartServicesMock = (
     ml: {
       locator,
     },
+    theme: {
+      theme$: {},
```

Comment:
Would it be worth making this:
```
theme$: themeServiceMock.createStartContract().theme$
```

Looks like the tests all passed and so maybe not necessary.

Reply:
This is much better I will update our mocks to include this. Thanks!!!